cmd/harmony/default.go

```diff
@@ -81,6 +84,10 @@ var defaultLogContext = logContext{
 	Port: 9000,
 }
 
+var defaultSysConfig = sysConfig{
+	NtpServer: "1.pool.ntp.org",
```

Comment:
Probably we can remove this variable since we may not need `getDefaultSysConfigCopy` (Check the other comment). Also, the value of NtpServer is defined twice in this package, which is not very good.

Reply:
removed